packages/quick_start/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "quick_start",
+    "version": "1.0.0",
+    "description": "Quick start Node sample app",
+    "main": "index.js",
+    "types": "index.d.ts",
+    "scripts": {
+        "build": "tsc",
+        "quick_start": "npx ts-node index.ts"
```

Comment:
You don't actually have index.ts, so change it to the file you do want - source/SampleApp.ts  , You might also need to fix the path it loads the json file